frontend/src/ui/Input.tsx

```diff
@@ -1,18 +1,15 @@
 import React, { useState } from "react";
 import { useTranslation } from "react-i18next";
 import { FiCheck, FiCopy } from "react-icons/fi";
+import { focusStyle } from ".";
 import { Button } from "./Button";
 import { WithTooltip } from "./Floating";
 
 
 const style = (error: boolean) => ({
     borderRadius: 4,
     border: `1px solid ${error ? "var(--danger-color)" : "var(--grey80)"}`,
-    "&:focus": {
-        outline: "none",
-        boxShadow: "0 0 0 1px var(--accent-color)",
-        borderColor: "var(--accent-color)",
-    },
+    ...focusStyle({}),
```

Comment:
With this, there is still a grey border right inside of the outline. I think it looks a bit off. I think I would keep the changed border color (although that makes the perceived outline thicker...). Or maybe make the border transparent? Mhhh